### PR TITLE
Editorial improvements

### PR DESCRIPTION
**Pull request by glyn**
Fixes https://github.com/ietf-wg-jsonpath/draft-ietf-jsonpath-base/issues/468

### CODE REVIEW

File: draft-ietf-jsonpath-base.md [hunk 1]
```diff
@@ -276,14 +272,18 @@ Unicode Scalar Value:
   In other words, integers in either of the inclusive base 16 ranges 0 to D7FF and
   E000 to 10FFFF.  Both JSON string values and JSONPath queries are sequences of Unicode scalar values.
 
-Singular Query:
-: A JSONPath expression built from segments each of which, regardless of the input value,
-  produces a nodelist containing at most one node.
+Segment:
+: One of the constructs which select children (`[<selectors>]`)
+  or descendants (`..[<selectors>]`) of an input value.
```
Comment (glyn) [inline draft-ietf-jsonpath-base.md:277]: Note to reviewers: I moved the updated definition of Segment down so that it is just before the definition of Selector. If I had left it in the previous position, I would have felt the need to add a reference from the definition of Segment to the definition of Selector, which I didn't want to get into.

File: draft-ietf-jsonpath-base.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 v: 3
 
-docname: draft-ietf-jsonpath-base-latest
+docname: draft-ietf-jsonpath-base-14
```
Comment (cabo) [inline draft-ietf-jsonpath-base.md:4]: This is probably not really part of the PR?

File: draft-ietf-jsonpath-base.md [hunk 3]
```diff
@@ -381,7 +381,7 @@ of the evaluation of a filter expression ({{filter-selector}}).
 
 ### Segments
 
-Segments select children (`[]`) or descendants (`..[]`) of an input value.
+Segments select children (`[<selectors>]`) or descendants (`..[<selectors>]`) of an input value.
```
Comment (cabo) [inline draft-ietf-jsonpath-base.md:384]: The only problem I have is that this amplifies a bit that segments are always bracket notation, which they aren't.

**Review by cabo — APPROVED** [review]

File: draft-ietf-jsonpath-base.md (hunk 2 above)
Comment (glyn) [reply draft-ietf-jsonpath-base.md:4]: Whoops! Reverted.

File: draft-ietf-jsonpath-base.md (hunk 3 above)
Comment (gregsdennis) [reply draft-ietf-jsonpath-base.md:384]: > segments are always bracket notation

I read this more that brackets are more of a formal notation.  Shorthand notation just works around that.

**Review by gregsdennis — APPROVED** [review]
This is fine with me.

File: draft-ietf-jsonpath-base.md (hunk 3 above)
Comment (cabo) [reply draft-ietf-jsonpath-base.md:384]: > I take your point, but actually the existing text does the same. We could add "e.g., " inside each of the parentheses, but that seems a bit fussy.

Indeed.  I just wanted to mention this observation, but it is not in the way of merging this PR.

**Comment by cabo** [comment]
Oh, and

```
399:Selectors are detailed in the next section. 
*** Run 'make fix-lint' to automatically fix some errors
draft-ietf-jsonpath-base.md contains trailing whitespace
make: *** [lib/main.mk:205: lint-whitespace] Error 1
```

Fixed.
